docs/the-button.md

```diff
@@ -131,3 +132,25 @@ A script that gets triggered every time The Button is pushed down. By default it
 The counter part of `down.sh` - it gets triggered every time The Button is released and by default it is stopping the LED flashing initiated in the down event.
 ### [`common.sh`](https://github.com/BlokasLabs/pisound/blob/master/scripts/common/common.sh)
 Defines APIs shared between scripts for Pisound, such as flash_leds and periodic_led_blink which are used by other scripts for feedback.
+
+## Command Line Options
+
+Current command line options are always available using the `--help` argument.  The options are:
+| Option | Description | 
+| --- | --- | 
+|`--help`| Display the usage information.|
+|`--version`| Show the version information. |
+|`--gpio <n>`| The pin GPIO number to use for the button. Default is 17. |
+|`--active-low`| Reverse the sense of the active state.  Normally active is when GPIO goes high |
+|`--conf <path>`| Specify the path to configuration file to use. Default is /etc/pisound.conf. |
+|`--click-count-limit <n>`| Set the click count limit to n. Use 0 for no limit. Default is 8.  This option will override the `CLICK_COUNT_LIMIT` configuration file entry.  The absolute limit is 99. |
+|`--no-defaults`| Do not use the default values for click and hold.  Only configuration options will be used.` |
+|--debug <n>`| Enable debugging at level n (higher value = more logging)
+|`-n <n>`| Short for --click-count-limit. |
+|`-q`| Short for --debug 0 (turns off all but errors) |
+
+<!--stackedit_data:
```

Comment:
--debug is missing a ` at the begining, the line before that has an extra one at the very end. Please terminate each line with a '.', so all of them are consistent.

What's stackedit_data? :)